### PR TITLE
@jscad/svg-serializer: convertColor: fix: opacity output

### PR DESCRIPTION
**Pull request by btakita**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

> Note: please do NOT include build files (those generate by the build-xxx commands) with your PR,

Thank you for your help in advance, much appreciated !

### CODE REVIEW

File: packages/io/svg-serializer/index.js [hunk 1]
```diff
@@ -203,7 +203,7 @@ const convertPath = (path, offsets, options) => {
   return str
 }
 
-const convertColor = (color) => `rgb(${color[0] * 255},${color[1] * 255},${color[2] * 255},${color[3] * 255})`
```
Comment (z3dev) [inline packages/io/svg-serializer/index.js:206]: I never noticed this before... but should 'rgb' become 'rgba'? I believe this should follow the specifications.

https://www.w3.org/TR/css-color-3/#rgba-color

**Review by z3dev — COMMENTED** [review]
@btakita Thanks for the nice fix. However, I think there may be another required change to follow the SVG/CSS specifications.

File: packages/io/svg-serializer/index.js (hunk 1 above)
Comment (z3dev) [reply packages/io/svg-serializer/index.js:206]: @btakita 

I just checked all browsers using rgba(r,g,b,a). All browsers rendered properly.

Please change rgb to rgba

**Review by z3dev — CHANGES_REQUESTED** [review]

**Comment by z3dev** [comment]
@btakita no change, no merge.

File: packages/io/svg-serializer/index.js (hunk 1 above)
Comment (btakita) [reply packages/io/svg-serializer/index.js:206]: Done

**Comment by btakita** [comment]
Hi @z3dev. Sorry, I lost track of this PR. I made the change.

**Review by z3dev — APPROVED** [review]

**Comment by z3dev** [comment]
@btakita Super! Thanks for following up, and making the changes.

Sadly, there was a recent release so these changes may take awhile to release. Are you looking for a quick release?

**Comment by btakita** [comment]
@z3dev I fixed the tests with the force push. I don't need a quick release, so no problem.

**Pull request merged**